sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCScanBuilder.scala

```diff
@@ -72,7 +72,7 @@ case class JDBCScanBuilder(
     if (!jdbcOptions.pushDownAggregate) return false
 
     val dialect = JdbcDialects.get(jdbcOptions.url)
-    val compiledAgg = JDBCRDD.compileAggregates(aggregation.aggregateExpressions, dialect)
+    val compiledAgg = aggregation.aggregateExpressions.flatMap(dialect.compileAggregate(_))
```

Comment:
This is wrong and is different from the previous behavior. We can't push down anything if one of the aggregate function is not supported.

Reply:
Done.